src/security/cert_generators/file/Makefile.am

```diff
@@ -25,6 +25,10 @@ security_file_certgen_SOURCES = \
 
 security_file_certgen_LDADD = \
 	$(top_builddir)/src/ssl/libsslutil.la \
+	$(top_builddir)/src/sbuf/libsbuf.la \
+	$(top_builddir)/src/error/liberror.la \
+	$(top_builddir)/src/mem/libminimal.la \
```

Comment:
AFAICT, Squid [fails](https://semaphoreci.com/squid/squid/branches/pull-request-1123/builds/3) to build on Semaphore CI because this backport PR is missing mem/libminimal from master/v6 commit 536a1f9. HTH.

@kinkie, AFAICT, Jenkins tests are currently badly broken because exactly the same build [failure](https://build.squid-cache.org/job/5-pr-test/4649/COMPILER=gcc,OS=debian-stable,label=docker-build-host/consoleFull) results in a _successful_ test outcome on Jenkins:

```
20:40:56 make[6]: *** No rule to make target '../../../../src/mem/libminimal.la', needed by 'security_file_certgen'.  Stop.
20:41:02 buildtest.sh result is 2
...
21:11:29 Build FAILED.
21:11:33 Finished: SUCCESS
```

Reply:
thanks